### ISSUE TITLE
More clear instructions

### ISSUE THREAD

**Issue by Amit0617** [vinceliuice/Orchis-theme#114]
```sudo snap install orchid-themes``` is not working, so we are required to install from github. So please give clear installation instructions for requirements part from terminal.

**Comment by vinceliuice** [comment]
sudo snap install orchis-themes

**Comment by vinceliuice** [comment]
not orchid

**Comment by Amit0617** [comment]
Oh! That was typo happened just here I checked the history I have tried "orchis-themes" only and even restarted the laptop but it haven't appeared on tweaks tool. So, that doesn't seems to do anything.

**Comment by vinceliuice** [comment]
Did you installed orchis theme before you install snap version？

**Comment by Amit0617** [comment]
No I simply ran that snap command I thought it would be simplest and fastest method to get the theme.

**Comment by vinceliuice** [comment]
Then Gnome-tweaks will not show the theme you only installed in snap

**Comment by Amit0617** [comment]
Then how can I activate it or apply on system?

**Comment by vinceliuice** [comment]
Run ./install.sh to install this theme

**Comment by Amit0617** [comment]
Oh! I was finding it on tweaks tool to apply and didn't tried ./install.sh
Thanks!😅

**Comment by Amit0617** [comment]
Excuse me where I will have to run ./install.sh command means where will be the directory when installed from snap?

**Comment by vinceliuice** [comment]
Nope, snap theme follow the normal system theme, if you don't have it in your system, you can not change your snap theme, there is not a app is made separately for snap themes
